locale/en_US/LC_MESSAGES/clr-installer.po

```diff
@@ -543,6 +543,33 @@ msgstr "Installation in progress."
 msgid "Installation successful."
 msgstr "Installation successful."
 
+msgid " (default)"
+msgstr " (default)"
+
+msgid "Add Extra Arguments:"
+msgstr "Add Extra Arguments:"
```

Comment:
Please drop the : from the label strings to be consistent with the User Add screen

Reply:
Done.